### ISSUE TITLE
[Feature Request]: npm-package supports multiple packages

### ISSUE THREAD

**Issue by eternasuno** [devcontainers-extra/features#82]
### Requested Feature

Right now npm-package can only install one package, so maybe there should be a new option for packages to install multiple packages, in the format `package1@version,package2@version`.

**Comment by ThePlenkov** [comment]
Indeed! If we look at the number of features - they're growing like mushrooms and everywhere is just 1 cli installed with npm. Why not just create one feature to support global packages and that's it?

Like we have for apt-get
```
ghcr.io/rocker-org/devcontainer-features/apt-packages:1": {
      "packages": ["libsecret-tools", "gnome-keyring"]
    },
```

**Comment by koralowiec** [comment]
Yeah, it makes sense. I think that I would prefer to a new feature `npm-packages` instead of extending `npm-package` though. I'm going to start working on the implementation

**Comment by koralowiec** [comment]
I added `npm-packages` in #104 

The packages can be defined in format proposed by @eternasuno: `package1@version,package2@version`
As far as I know, it's not possible to define a feature option as an array currently (https://github.com/devcontainers/spec/issues/57)

The feature will be available once it's published by [workflow](https://github.com/devcontainers-extra/features/actions/runs/13472664940)

**Comment by ThePlenkov** [comment]
@koralowiec  it seems like it doesn't work for packages with namespace. Like I tried with `@sourcegraph/cody` and it doesn't work

**Comment by ThePlenkov** [comment]
Created #110 PR
